.travis.yml

```diff
@@ -2,8 +2,10 @@ language: crystal
 
 before_install:
   - sudo rm -vf /etc/apt/sources.list.d/*riak*
+  - sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
+  - echo "deb http://download.mono-project.com/repo/ubuntu trusty main" | sudo tee /etc/apt/sources.list.d/mono-official.list
   - sudo apt-get -qq update
-  - sudo apt-get install -y default-jdk haskell-platform
+  - sudo apt-get install -y default-jdk haskell-platform mono-devel
```

Comment:
Why not just install `mono-mcs` from default apt sources?

Reply:
I just follow the instructions from official site to install mono. I can change it to mono-mcs.